src/Microsoft.AspNetCore.Razor.Runtime/TagHelpers/ITagHelperComponent.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Razor.TagHelpers
+{
+    /// <summary>
+    /// Contract used to filter matching HTML elements.
```

Comment:
@NTaylorMullen Why should I nuke the doc comment for `ITagHelper` then? https://github.com/aspnet/Razor/pull/1096#discussion_r107040990